### DISCUSSION TITLE
Import of a specific month with GoCardless fails without clear error

### DISCUSSION THREAD

**Discussion by filip-van-hoeckel** [firefly-iii/firefly-iii#9564] [Firefly III]
```
Error(s) from the import process
Line #0: No transactions were downloaded from GoCardless. You may be rate limited or something else went wrong.
```
One way or the other I've been struggling to get this to even import a month of data from 1(!) account. I have access to the logs, but no pointers as to what to look for in order to pinpoint the issue. Please tell me what to look for, thanks.

**Comment by JC5** [comment]
The logs should report what comes back from GoCardless, that could be a hint as to what's going on.
